daemon/CMakeLists.txt

```
@@ -1,12 +1,21 @@
+if(APPLE)
+  set(WITH_BOOTSTRAP OFF)
+else()
+  option(WITH_BOOTSTRAP "build lokinet-bootstrap tool" ON)
+endif()
```

```suggestion
set(DEFAULT_WITH_BOOTSTRAP ON)
if(APPLE)
  set(DEFAULT_WITH_BOOTSTRAP OFF)
endif()
option(WITH_BOOTSTRAP "build lokinet-bootstrap tool" ${DEFAULT_WITH_BOOTSTRAP})
```
so that you can still build it on Apple if you explicitly enable it.